java/src/jmri/jmrit/powerpanel/PowerPane.java

```diff
@@ -82,10 +81,16 @@ protected void choiceChanged() {
         idleButton.setToolTipText(Bundle.getMessage("ToolTipIdleButton"));
 
         // general GUI config
-        setLayout(new jmri.util.javaworld.GridLayout2(3, 2, 6, 0)); // r, c, hgap , vgap
+        setLayout(new java.awt.GridLayout(3, 2, 0, 5)); // r, c, hgap , vgap
```

Comment:
I would appreciate some hgap. There is no space between the label "Unknown" and the button "Off".